python/lsst/pipe/tasks/postprocess.py

```diff
@@ -789,6 +791,167 @@ def _makeArgumentParser(cls):
+        schema.addField('ccd', type='I', doc='Detector number')
+        schema.addField('filterName', type='String', size=20, doc='Filter name')
+        schema.addField('psfSigma', type='F',
+                        doc='PSF model full width half-maximum (center of chip)')
+        schema.addField('psfArea', type='F',
+                        doc='PSF model effective area (center of chip)')
+        schema.addField('psfIxx', type='F',
+                        doc='PSF model Ixx (center of chip)')
+        schema.addField('psfIyy', type='F',
+                        doc='PSF model Iyy (center of chip)')
+        schema.addField('psfIxy', type='F',
+                        doc='PSF model Ixy (center of chip)')
+        schema.addField('raCorners', type='ArrayD', size=4,
+                        doc='Right Ascension of bounding box corners')
+        schema.addField('decCorners', type='ArrayD', size=4,
+                        doc='Declination of bounding box corners')
+
+        cat = afwTable.ExposureCatalog(schema)
+        cat.resize(len(dataRefs))
+
+        cat['visit'] = visit
+
+        if not self.isGen3:
+            bbox = lsst.geom.BoxI(lsst.geom.PointI(0, 0), lsst.geom.PointI(1, 1))
+
+        for i, dataRef in enumerate(dataRefs):
+            if self.isGen3:
+                visitInfo = dataRef.get(component='visitInfo')
+                f = dataRef.get(component='filter')
+                psf = dataRef.get(component='psf')
+                wcs = dataRef.get(component='wcs')
+                photoCalib = dataRef.get(component='photoCalib')
+                detector = dataRef.get(component='detector')
+                bbox = dataRef.get(component='bbox')
+            else:
+                exp = dataRef.get(datasetType='calexp_sub', bbox=bbox)
+                visitInfo = exp.getInfo().getVisitInfo()
+                f = exp.getFilter()
+                psf = exp.getPsf()
+                wcs = exp.getWcs()
+                photoCalib = exp.getPhotoCalib()
+                detector = exp.getDetector()
+                bbox = dataRef.get(datasetType='calexp_bbox')
+
+            rec = cat[i]
+            rec.setBBox(bbox)
+            rec.setVisitInfo(visitInfo)
+            rec.setWcs(wcs)
+            rec.setPhotoCalib(photoCalib)
+
+            rec['filterName'] = f.getName()
+            rec['ccd'] = detector.getId()
+            shape = psf.computeShape(bbox.getCenter())
+            rec['psfSigma'] = shape.getDeterminantRadius()
+            rec['psfIxx'] = shape.getIxx()
+            rec['psfIyy'] = shape.getIyy()
+            rec['psfIxy'] = shape.getIxy()
+            im = psf.computeKernelImage(bbox.getCenter())
+            rec['psfArea'] = np.sum(im.array)/np.sum(im.array**2.)
+
+            sph_pts = wcs.pixelToSky(lsst.geom.Box2D(bbox).getCorners())
+            rec['raCorners'][:] = [sph.getRa().asDegrees() for sph in sph_pts]
+            rec['decCorners'][:] = [sph.getDec().asDegrees() for sph in sph_pts]
```

Comment:
Ah, degrees. If we can't use an inherently unit-full storage type, I'm torn on whether degrees or radians is most appropriate here.